installing/installing_bare_metal/installing-bare-metal.adoc

```diff
@@ -32,7 +32,7 @@ include::modules/cluster-entitlements.adoc[leveloffset=+1]
 
 .Additional resources
```

Comment:
It's convoluted but in cases such as this one, we don't use a unique ID because we want it to display as a "[trailing include](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#:~:text=Only%20use%20.%20syntax%20(.Additional%20resources)%20to%20follow%20the%20module%20as%20a%20trailing%20include%20at%20the%20assembly%20level%2C%20which%20indicates%20that%20the%20resources%20apply%20specifically%20to%20the%20module.)" that relates to the preceding module rather than to the assembly as a whole.

> **Note**
> When using "Prerequisites", "Next steps", and "Additional resources" headings in an assembly, use == formatting, such as == Prerequisites and == Additional resources. Use of this heading syntax at the assembly level indicates that the sections relate to the whole assembly. Only use . syntax (.Additional resources) to follow the module as a trailing include at the assembly level, which indicates that the resources apply specifically to the module.